v03_pipeline/lib/tasks/update_project_table.py

```diff
@@ -28,7 +29,13 @@ def complete(self) -> bool:
             and super().complete()
             and hl.eval(
                 hl.read_table(self.output().path).updates.contains(
-                    self.callset_path,
+                    hl.Struct(
```

Comment:
Another possible approach here is to store the set of families loaded for each callset on the globals.  This works, but it way more storage and annoyance.

Reply:
We also cannot just check that the families in the pedigree are already in the project table because they may have been loaded from a previous callset!

Reply:
I like the hash approach you did, its functionally checking the same thing as storing the full families/samples list but easier to maintain